addons/mrp/static/src/mrp_forecasted/forecasted_details.js

```diff
@@ -0,0 +1,12 @@
+/** @odoo-module **/
+
+import { ForecastedDetails } from '@stock/stock_forecasted/forecasted_details';
+import { StockForecasted } from '@stock/stock_forecasted/stock_forecasted';
+
+export default class MRPForecastedDetails extends ForecastedDetails {
+
+    displayReserve(line) {
+        return line.move_out && (line.move_out.picking_id || line.move_out.raw_material_production_id) && !line.in_transit;
```

Comment:
I think it should be better to call super instead of rewriting the all the condition
```suggestion
        return super(...arguments ) || (line.move_out?.raw_material_production_id && !line.in_transit);
```

Or another solution (I prefer but a little bit more changes):
- In `stock` `ForecastedDetails`
```javascript
displayReserve(line) {
    return !line.in_transit && this.reservingOperation ;
}

...

get reservingOperation() {
    return line.move_out?.picking_id;
}
```
- then here:
```javascript
get reservingOperation() {
    return super() || line.move_out?.raw_material_production_id;
}
```